lib/tasks/accounts/update_adopter_status.rake

```diff
@@ -0,0 +1,28 @@
+namespace :accounts do
+  desc 'Update newly created field for adopter status from Salesforce'
+  # rake accounts:update_adopter_status
+  task update_adopter_status: [:environment] do
+    loop do
+      users = User.where(adopter_status: nil).where.not(salesforce_contact_id: nil ).limit(250)
+
+      contacts = OpenStax::Salesforce::Remote::Contact.select(
+        :id,
+        :adoption_status,
+        :accounts_uuid
+      )
+      .where(id: users.map(&:salesforce_contact_id))
+      .index_by(&:id)
```

Comment:
@Dantemss it looks like `index_by` is not supported by ActiveForce. I'll see what I can do to fix this.